app/converse-plugins/desktop-credentials.js

```diff
@@ -2,17 +2,19 @@ const credentials = await import('../credentials.js');
 
 converse.plugins.add('converse-desktop-credentials', {
 
-    initialize () {
-        const { _converse } = this;
-        const { api } = _converse;
```

Comment:
Not a big deal, but I'd prefer if we followed the Converse style-guide in this repo as well.

See here: https://conversejs.org/docs/html/style_guide.html

There isn't anything specific to destructuring in the style-guide, but in the converse.js code-base we have spaces in a destructured assignment (e.g. `const { api } = _converse`).

And the other main thing is spaces after a function name in a function declaration (e.g. `initialize () { ... }`).

Thanks

Reply:
Alright, I will see if I can get Webstorm's formatting configured accordingly and update.